<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiConfigTab</class>
 <widget class="QWidget" name="GuiConfigTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="reverseDragBehaviorCheck">
     <property name="text">
      <string>Use &amp;left mouse button to drag and change center frequency</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="noLimitsCheck">
     <property name="text">
      <string>Ignore &amp;hardware limits (use with caution)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="useGLWaterfallCheck">
     <property name="text">
      <string>Enable Open&amp;GL-based waterfall (needs restart, experimental)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="useMaxBlendingCheck">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Enable &amp;max blending to downscale waterfall data</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="useGlWfInWindowsCheck">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Enable OpenGL waterfall in &amp;detached inspector windows</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="ttlCheck">
     <property name="text">
      <string>Allow GUI to &amp;discard spectrum updates under heavy load</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="ttlLabel">
     <property name="text">
      <string>Max TTL for spectrum updates</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="ttlSpin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Overlay spectrum informative text</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="3" column="0" colspan="2">
       <widget class="QPlainTextEdit" name="infoTextEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string extracomment="Type informative text here...">Type informative text here...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="ColorChooserButton" name="infoTextColor" native="true">
        <property name="color" stdset="0">
         <color>
          <red>85</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Text color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorChooserButton</class>
   <extends>QWidget</extends>
   <header>ColorChooserButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
